### PR TITLE
workaround for issue #265

### PR DESCRIPTION
**Pull request by fxha**
| Q                | A
| ---------------- | ---
| Bug fix?         | yes
| Fixed tickets    | #265
| License          | MIT

### Description

Validate `theme` and `bulletListMarker` preferences.

### CODE REVIEW

**Comment by Jocs** [comment]
@fxha 

We don't need to go too far here, after all, after the 1.0 release, we will develop a graphical setup interface that will be verified at the front end.

So just play around! 👍

File: src/main/preference.js [hunk 1]
```diff
@@ -93,6 +94,32 @@ class Preference {
       }
     })
   }
+
```
Comment (Jocs) [inline src/main/preference.js]: Hi, @fxha we can use [`json schema`](http://json-schema.org/) to valid user setting! but I think we can put it after version 1.0.0(or you want to do before version 1.0.0). So you can add a TODO?

webpack used [ajv](https://github.com/epoberezkin/ajv)

**Comment by fxha** [comment]
That's why I only check theme (and bulletListMarker) because invalid theme values can crash the renderer process. I do not do any more verification.

**Comment by Jocs** [comment]
@fxha we can also use `json schema` to validate the user setting at back end too.

**Comment by Jocs** [comment]
@fxha If you want to use `json schema` to validate user setting, you can add it to the next project, I think it necessary before `graphical setup interface`.

So please let me know weather you want to do this job? So I can decide weather to merge this PR.(If you want to use json schema, You can open another PR or update this PR?)

**Comment by fxha** [comment]
I'll create an issue and add it to the next project. So you can merge this.

**Comment by Jocs** [comment]
@fxha ok

**Pull request merged**